### ISSUE TITLE
Escaped API response causes pydantic max_length validation to fail

### ISSUE THREAD

**Issue by ilaskaev** [juftin/lunchable#96]
-   lunchable version: 0.9.0
-   Python version: 3.11

### Description

Lunchmoney API (`/v1/categories/group/{category_id}/add` in this case) sometimes seems to send back some escaped characters (eg. `&#x2F;` instead of `/`), which results in pydantic `max_length` validations to sometimes fail as string length is not counted correctly.

![image](https://github.com/juftin/lunchable/assets/16270230/69e3f6df-7238-45f9-b49c-692508c9c42b)

### What I Did

1. Create a new category that is of 40 characters long and add it to a category group
2. Add a new category in same category group with `insert_into_category_group` method
3. Error will be raised as the `children[]` will fail instantiation due to having a length > 40 characters.

**Comment by juftin** [comment]
Hey @ilaskaev thanks for reporting 🙇 . That's a bummer, I'll look at only validating for 40 characters on the create / update  side instead of the read side. 

There are a number of big changes coming, including Pydantic v2 - I should be able to get this resolved in the next few days hopefully.

**Comment by juftin** [comment]
This is a bug being tracked here https://feedback.lunchmoney.app/bugs/p/unescape-category-names-in-api-response

FYI @ilaskaev - are you okay with waiting for that to be resolved or would you like a resolution in lunchable?

**Comment by ilaskaev** [comment]
Yep not a problem, I monkey patched the issue from my side to make things work.
